docs/concepts/custom-itemspanel.md

```diff
@@ -0,0 +1,71 @@
+---
+description: CONCEPTS
+---
+
+import exampleScreenshot from '/img/concepts/customitemspanel/ItemsControl-with-ItemPanel-and-Styles.png';
+
+# Custom ItemsPanel
+
+the following built-in controls, display their items in a StackPanel by default: 
```

Comment:
This is not true for all of these panels. For example Carousel uses it's own special custom panel

https://github.com/thevortexcloud/Avalonia/blob/2a7b5568b7f61fe6df9c46a05bf1c9d474c9290e/src/Avalonia.Controls/Carousel.cs#L18-L23